### PR TITLE
Add Shopify docs

### PR DESCRIPTION
**Pull request by dannytranlx**
@mspensieri @jpjoyal @kvgauthier

### CODE REVIEW

File: source/includes/integrations/_shopify.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+This integration adds the Smooch web plugin to your Shopify online store. We will take care of setting your `appToken` and adding the code snippet at the right place in your shop theme.
+
+### Adding Smooch to your store
+
+In order to append the plugin to your theme, all you need to do is to press on _Authenticate_ from our integration page or install our app from the [Shopify marketplace](https://apps.shopify.com/smooch).
+
+![From the integration page](/images/shopify.png)
+
+<aside class="notice">Integrating Smooch in your store is theme-based. You will need to repeat the process to re-add the widget to your new theme.</aside>
```
Comment (mspensieri) [inline source/includes/integrations/_shopify.md]: What does this mean?

Comment (dannytranlx) [inline source/includes/integrations/_shopify.md]: In technical terms, I get yo theme layout and I add `our widget snippet` right before `</body>`. But if you change your theme to a new one, it won't be there anymore.

Comment (mspensieri) [inline source/includes/integrations/_shopify.md]: So you need to de-integrate, then re-integrate, right?

Comment (dannytranlx) [inline source/includes/integrations/_shopify.md]: Yes

Comment (jpjoyal) [inline source/includes/integrations/_shopify.md]: no. Suggest: "Changing your theme after enabling this integration might remove it, you'll need to re-enabled this integration in this case"

Comment (mspensieri) [inline source/includes/integrations/_shopify.md]: > Integrating Smooch will modify your current Shopify theme. If your theme ever changes, you will need to reintegrate.

?

File: source/includes/integrations/_shopify.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Shopify
+
+[Shopify](https://www.shopify.com/) is a simple way to set up an online store to organize products, accept credit card payments, and manage orders.
+
+This integration adds the Smooch web plugin to your Shopify online store. We will take care of setting your `appToken` and adding the code snippet at the right place in your shop theme.
```
Comment (kvgauthier) [inline source/includes/integrations/_shopify.md]: We avoid saying we and use Smooch instead. Also adjusting verb tense:
Smooch will set your `appToken` and add the code snippet at the right location in your shop theme.

**Comment by jugarrit** [comment]
:+1:

File: source/includes/integrations/_shopify.md (hunk 2 above)
Comment (dannytranlx) [inline source/includes/integrations/_shopify.md]: We need docs to know what style to write in :disappointed:

File: source/includes/integrations/_shopify.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+### Adding Smooch to your store
+
+In order to append the plugin to your theme, all you need to do is to press on _Authenticate_ from our integration page or install our app from the [Shopify marketplace](https://apps.shopify.com/smooch).
```
Comment (kvgauthier) [inline source/includes/integrations/_shopify.md]: *from the integration page or install Smooch from the ...

Comment (dannytranlx) [inline source/includes/integrations/_shopify.md]: Done

**Pull request merged**